packages/core/src/lib/lit-core.ts

```diff
@@ -477,6 +467,9 @@ export class LitCore {
   }
 
   private async _connect() {
+    if (!this.ready) {
```

Comment:
Can we get rid of this entirely now that the methods call `initWasm()` internally when they need to?
Can we update `checkSevSnpAttestation()` to call the new wrapper methods that init it for us instead?

Reply:
see above comment. if we change the `crypto` apis we will be making breaking changes.